lib/Parser/Scan.cpp

```diff
@@ -1552,7 +1460,7 @@ tokens Scanner<EncodingPolicy>::ScanStringConstant(OLECHAR delim, EncodedCharPtr
 LBreak:
     bool createPid = true;
 
-    if (m_fSyntaxColor || (m_DeferredParseFlags & ScanFlagSuppressStrPid) != 0)
+    if ((m_DeferredParseFlags & ScanFlagSuppressStrPid) != 0)
```

Comment:
I think ScanFlagSuppressStrPid can be deleted. I thought background parsing was using it, but I'm thinking of the ScanNoKeywords path.l